### PR TITLE
[DX-1437] centre --> center

### PR DESCRIPTION
**Pull request by Roeegg2**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/developer-support/upgrading-tyk/deployment-model/self-managed/overview.md
Comment (letzya) [inline tyk-docs/content/developer-support/upgrading-tyk/deployment-model/self-managed/overview.md:1]: no anchor link in use

File: tyk-docs/content/tyk-on-premises/licensing.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Do you need support for centrally managing your APIs while deploying edge gatewa
 
 {{< img src="/img/diagrams/tyk-selfmanaged-launch.png" alt="Tyk Self-managed Launch">}}
 
-## Flex - for multiple teams within a single data centre or cloud 
+## Flex - for multiple teams within a single data center or cloud
```
Comment (letzya) [inline tyk-docs/content/tyk-on-premises/licensing.md:50]: We need to update this rel link https://github.com/TykTechnologies/tyk-docs/blob/940e95b05713279c328be16ecaf2fbc6d06cd39e/tyk-docs/content/tyk-on-premises/licensing.md#L20

**Review by letzya — CHANGES_REQUESTED** [review]
Update https://github.com/TykTechnologies/tyk-docs/blob/940e95b05713279c328be16ecaf2fbc6d06cd39e/tyk-docs/content/tyk-on-premises/licensing.md#L20

**Comment by letzya** [comment]
@dcs3spp please don't merge. Need to get this approved
Update: Approved.

**Comment by dcs3spp** [comment]
@letzya Can this be [merged](https://github.com/TykTechnologies/tyk-docs/pull/5012#issuecomment-2210864885) now?

**Review by letzya — DISMISSED** [review]

**Review by letzya — APPROVED** [review]

**Pull request merged**